runtime/src/main/java/org/corfudb/runtime/CheckpointWriter.java

```diff
@@ -126,8 +126,11 @@ public CheckpointWriter(CorfuRuntime rt, UUID streamId, String author, T map) {
      */
     public Token appendCheckpoint() {
         long start = System.currentTimeMillis();
+        Token markerToken = rt.getSequencerView().next().getToken();
+        rt.getAddressSpaceView().write(markerToken, "marker".getBytes());
```

Comment:
Please use a constant for `"marker".getBytes()`

Reply:
Done.